internal/stream_fetch_helper.go

```diff
@@ -27,7 +27,6 @@ type LogFetchHandlers interface {
 	AfterFetch(logs []storage.Object) error
 }
 
-// TODO : unit tests
 func ParseTS(endTSEnvVar string) (endTS *time.Time, err error) {
 	endTSStr, ok := GetSetting(endTSEnvVar)
 	if ok {
```

Comment:
If there is no such environment variable, the function will return nil instead of an error.

Reply:
Yes, and I test it in TestParseTs_shouldReturnNilOnNoTime. Is something wrong with that?